src/dev-app/slide-toggle/slide-toggle-demo.html

```diff
@@ -1,29 +1,8 @@
-<div class="demo-slide-toggle">
-
-  <mat-slide-toggle color="primary" [(ngModel)]="firstToggle">
-    Default Slide Toggle
-  </mat-slide-toggle>
-
-  <mat-slide-toggle [(ngModel)]="firstToggle" disabled>
-    Disabled Slide Toggle
-  </mat-slide-toggle>
-
-  <mat-slide-toggle [disabled]="firstToggle">
-    Disable Bound
-  </mat-slide-toggle>
-
-  <p>Example where the slide toggle is required inside of a form.</p>
-
-  <form #form="ngForm" (ngSubmit)="onFormSubmit()" ngNativeValidate>
-
-    <mat-slide-toggle name="slideToggle" required ngModel>
-      Slide Toggle
-    </mat-slide-toggle>
-
-    <p>
-      <button mat-raised-button type="submit">Submit Form</button>
-    </p>
-
-  </form>
-
-</div>
+<h3>Slide Toggle Overview</h3>
+<slide-toggle-overview-example></slide-toggle-overview-example>
+<h3>Configurable Slide Toggle</h3>
```

Comment:
Can we keep the old inline demo? I use the dev app for debugging and these demos that are based on the docs examples tend to be annoying to debug. We already have the `/examples` page where all the examples can be seen.

Reply:
Sure, I'll do that then request review again.